enforcer-rules/src/main/java/com/google/cloud/tools/opensource/enforcer/LinkageCheckerRule.java

```diff
@@ -110,24 +125,41 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
       try {
         LinkageChecker linkageChecker = LinkageChecker.create(classpath, directDependencies);
         LinkageCheckReport linkageReport = linkageChecker.findLinkageErrors();
-        int totalErrors =
+        long errorCount =
```

Comment:
Why long? There shouldn't be that many, and more importantly we're mapping to int below.